tests/Microsoft.Bot.Builder.Azure.Tests/BlobsTranscriptStoreTests.cs

```diff
@@ -4,8 +4,12 @@
 using System;
```

Comment:
Note: Tests were changed here and not `TranscriptStoreBaseTests` because overwrite behavior differs between `BlobsTranscriptStore` and `AzureBlobTranscriptStore`.